src/components/Pokemon.js

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+function Pokemon({pokemon}) {
+
+    const [pokemonData, setPokemonData] = useState({});
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        async function fetchData() {
+            setError('');
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemon}`);
+                setPokemonData(result.data);
+            } catch (e) {
+                console.error(e);
+                setError("er is iets mis gegaan bij het ophalen van de data. Probeer opnieuw.");
+            }
+        }
+        fetchData();
+    }, []);
+
+
+    function showAbility(array) {
+        let newArray = array.map((abilities) => {
+            return <li key={abilities.ability.name}>{abilities.ability.name}</li>;
+        });
+        return newArray;
+    }
+
+    return (
+        <>
+            {error && <p>{error}</p>}
+
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Als je de default waarde van `pokemonData` undefined hebt gemaakt, dan kun je hier het volgende doen:
```suggestion
            {pokemonData &&
```